tests/unit/pywbemcli/simple_interop_mock_script.py

```diff
@@ -57,17 +54,28 @@ def _setup(conn, server, verbose):
         this_file_path = 'tests/unit/pywbemcli/simple_interop_mock_script.py'
         assert os.path.exists(this_file_path)
 
-    mof_file = 'simple_interop_mock_model.mof'
-    dependent_files = [mof_file,
-                       'mock_interop.mof',
-                       'simple_mock_model.mof']
-    mof_path = os.path.join(os.path.dirname(this_file_path), mof_file)
-    conn.compile_mof_file(mof_path, namespace=None)
+    # Prepare an Interop namespace and namespace provider
+
+    interop_mof_file = 'mock_interop.mof'
```

Comment:
Same comment about packaging this as method in pywbemcli

Reply:
see above